apps/wallet-mobile/src/features/RampOnOff/useCases/CreateExchange/ShowDisclaimer/ShowDisclaimer.tsx

```diff
@@ -25,26 +26,29 @@ export const ShowDisclaimer = () => {
   )
 }
 
-const styles = StyleSheet.create({
-  container: {
-    paddingHorizontal: 16,
-    paddingVertical: 12,
-  },
-  gradient: {
-    opacity: 1,
-    borderRadius: 8,
-    marginTop: 24,
-  },
-  title: {
-    fontSize: 16,
-    fontFamily: 'Rubik',
-    fontWeight: '500',
-    lineHeight: 24,
-  },
-  text: {
-    fontSize: 14,
-    fontFamily: 'Rubik',
-    lineHeight: 22,
-    marginTop: 8,
-  },
-})
+const useStyles = () => {
+  const {theme} = useTheme()
+  const styles = StyleSheet.create({
+    container: {
+      paddingHorizontal: 16,
+      paddingVertical: 12,
+    },
+    gradient: {
+      opacity: 1,
+      borderRadius: 8,
+      marginTop: 24,
+    },
+    title: {
+      ...theme.typography['body-1-regular'],
+      color: theme.color.gray.max,
+      fontWeight: '500',
```

Comment:
@SorinC6 can you check with Design, I really don't like the idea of having the Typography set, and still need to add `fontWeight` or whatever other property to the fonts that is not pointing to the typography directly.

Reply:
@stackchain Have looked into it before but was not sure. To have the fontWeight in the theme we need to introduce something like this: `body-1-regular-500` ,  `body-1-regular-400` , and `body-1-regular-600`,  for most of the theme text typography variants

Reply:
hmm, @SorinC6 my point is, `body-<1>` there is zero semantic meaning of this `1` what does it represent? a combination of what? mode + weight? only weight? only size? Let's arrange a call with Design, for me a good semantic would be `body-small-regular` + `strong`, this clearly indicates that fontWeight is not part of the first token, and I'd like to have a consistency around `small` `normal` `big`, we should leverage the fontScale, cuz some devices folks bump up a lot the their fonts, therefore, small, normal, big, whatever, should be a `base * fontScale * <small | normal | big>`

Reply:
@stackchain I've been following the pattern as h1,h2,h3... as on the design are\
  ```
'heading-1-regular': {
    ...fontFamily.regular,
    ...size.h1,
  },
  'heading-2-medium': {
    ...fontFamily.medium,
    ...size.h2,
  },
  'heading-2-regular': {
    ...fontFamily.regular,
    ...size.h2,
  },
  'heading-3-medium': {
    ...fontFamily.medium,
    ...size.h3,
  },
  'heading-3-regular': {
    ...fontFamily.regular,
    ...size.h3,
  },
```

And for the body adding the same pattern 🤔

```
  body1: {
    fontSize: 16,
    lineHeight: 24,
  },
  body2: {
    fontSize: 14,
    lineHeight: 22,
  },
  body3: {
    fontSize: 12,
    lineHeight: 18,
  },
```

Then adding this 
```

  'body-1-medium': {
    ...fontFamily.medium,
    ...size.body1,
  },
  'body-1-regular': {
    ...fontFamily.regular,
    ...size.body1,
  },
  'body-2-medium': {
    ...fontFamily.medium,
    ...size.body2,
  },
```

for the heading I think we can keep it as 1 ,2,3.. as above and the body we can change as: 
- body-small-medium-(fontWeight) , body-normal-medium-(fontWeight), body-big-medium-(fontWeight), body-small-regular-(fontWeight), body-normal-regular-(fontWeight), body-big-regular--(fontWeight)